yt/frontends/enzo/fields.py

```diff
@@ -291,8 +291,18 @@ def _tot_minus_kin(field, data):
                 function=_tot_minus_kin,
                 units=unit_system["specific_energy"])
         if multi_species == 0 and 'Mu' in params:
+            def _mean_molecular_weight(field, data):
+                return params["Mu"]*data['index', 'ones']
+
+            self.add_field(
+                ("gas", "mean_molecular_weight"),
+                sampling_type="cell",
+                function=_mean_molecular_weight,
+                units=unit_system["number_density"])
```

Comment:
Are these the correct units?

Reply:
No, you are right, I'll fix that. Thanks for the catch.